core/templates/core/login.html

```diff
@@ -0,0 +1,39 @@
+{% extends 'core/base.html' %}
+
+{% block title %}Logowanie{% endblock %}
+
+{% block content %}
+<script type="module">
+    const canvas = document.getElementById('starback-bg');
+    const starback = new Starback(canvas, {
+        type: 'dot',
+        quantity: 250,
+        height: window.innerHeight,
+        width: window.innerWidth,
+        backgroundColor: [],
+        randomOpacity: true,
+    });
+    function resizeCanvas() {
+        canvas.width = window.innerWidth;
+        canvas.height = window.innerHeight;
+    }
+
+    window.addEventListener('resize', resizeCanvas);
+</script>
+<canvas id="starback-bg" class="fixed -z-1" width="871" height="911"></canvas>
+<div class="flex justify-center grid justify-items-center bg-transparent place-content-center">
```

Comment:
1. Why do you use flex and grid, not only in one document, but in one element??
2. Add h-[80vh], so it doesn't overflow
3. This bg-transparent is useless

Reply:
1. I was learning
3. Because it had so many designs that it just stayed there